cohttp/src/link.ml

```diff
@@ -382,7 +382,7 @@ let rec params_of_string s i ps =
           let charset, language, v, i = star_of_string s i in
           params_of_string s i
             (Star { Ext.charset; language; value = Link_extension (main, v) }
-             :: ps)
+            :: ps)
```

Comment:
Can we also leave this as it was?

Reply:
This was the original formatting (pre-`ocamlformat.0.17.0`), that has been broken in 0.17.0 then fixed back in `ocamlformat.0.19.0`

Reply:
Oh… Fine then, even though like the broken one better :P